app/views/partners/profiles/edit/_area_served.html.erb

```diff
@@ -0,0 +1,38 @@
+<section class="content">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-md-12">
+        <div class="card">
+          <div class="card-header bg-white">
+            <h3 class="card-title"><strong>Area Served  (County / Client Share %) </strong></h3>
+          </div>
+          <div class="card-body">
+
+            <fieldset style="margin-bottom: 2rem">
+              <legend></legend>
+              <div id="partner-counties"  class="partner-county-fields partner-counties">
```

Comment:
```suggestion
              <div id="partner_counties"  class="partner-county-fields partner-counties">
```
The id of defined in the link_to_new_association was different from this one which was preventing it from adding the association to the right spot.